2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,6 +18,7 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
+  return Object.fromEntries(countryCurrencyCodes);
```

Comment:
nice use of this object method; not going to lie I didn't actually know this one existed to create objects from key/value pair array of arrays